app/controllers/organizations_controller.rb

```diff
@@ -27,7 +27,7 @@ def connect_lightspeed
   def create
     @organization = Organization.new(permitted_create_params)
     if @organization.save
-      membership = Membership.create(user_id: current_user.id, role: "admin", organization_id: @organization.id)
+      _membership = Membership.create(user_id: current_user.id, role: "admin", organization_id: @organization.id)
```

Comment:
I'd prefer just removing this assignment, rather than using an underscore variable.

Reply:
Good catch on seeing that it wasn't necessary though!